layouts/shortcodes/dbm-mongodb-before-you-begin.md

```diff
@@ -0,0 +1,13 @@
+Supported Agent versions
+: 7.55.0+
+
+Performance impact
+: The default Agent configuration for Database Monitoring is conservative, but you can adjust settings such as the collection interval and operation sampling rate to better suit your needs. For most workloads, the Agent represents less than one percent of query execution time on the database and less than one percent of CPU. <br/><br/>
+
+Connection strings or SRV strings
+: Although MongoDB connection strings or SRV strings provide many benefits such as automatic failover and load balancing, the Datadog Agent must connect directly to the MongoDB instance being monitored. If the Agent connects to different MongoDB instance while it is running (as in the case of failover, load balancing, and so on), the Agent calculates the difference in statistics between two hosts, producing inaccurate metrics. <br/><br/>
```

Comment:
How painful is this experience for users, do we know if mongo reuses addresses when they have a failover or will these be constantly changing? Will users have to configure their monitoring every month?

Reply:
i have not seen underlying hostname changes. A node usually comes back with same address. in self hosted if the underlying linux instance died and replaced, manual reconfigure in replica set is required.

Reply:
kk, but this would be fixed by the clusters autodiscovery thing we could do right?

Reply:
yes